Assets/Scripts/Basic Player Movement/CharacterController.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+
+public class CharacterController : MonoBehaviour
```

Comment:
Possibly having "Move" or "Movement" in the class name may have been more descriptive, since it's controlling the player movement. Not fussed though since this is a throwaway script but worth thinking about

Reply:
my original was "basic player movement" but that was the name the AI gave the script that I ended up basing it on and thought it would be less hassle to keep the name